openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -637,6 +671,13 @@ def get_template_path(self):
             ).format(task_name, task_type, host_name))
 
         path = profile["path"]
+
+        # switch to remove placeholders after they are used
+        placeholder_keep = profile.get("placeholder_keep")
+        # backward compatibility, since default is True
+        if placeholder_keep is not False:
```

Comment:
```suggestion
        if placeholder_keep is None:
```

Reply:
Why it was resoled? Using eyplicit check instead of double negative is valid comment.

Reply:
Resolved by mistake ;)